### ISSUE TITLE
AMPlified Pages with Oxygen for Mobile Devices

### ISSUE THREAD

**Issue by WrongGuy** [soflyy/oxygen-bugs-and-features#201]
**Describe the feature you'd like to see included in Oxygen.**
*So far Oxygen is by far the fastest on earth, I want to see AMPlified Pages created with Oxygen. For Only mobile view either user or it will add/change the AMP Tags.*

**What are the use cases for this feature?**
*This will help the mobile results in Google to show AMP Sign with result and obviously load faster. For mobile pages all it will do is add the AMP Tags in the divs or sections.*

**Examples of this feature or functionality.**
*https://www.ampproject.org/docs/getting_started/create/basic_markup*

**Comment by teddarling** [comment]
Would love this myself. That said, I just noticed today that Oxygen pages at least look the same when using the official AMP plugin for WP. I can't say that for any of the other page builders that I've looked at.

I guess basically at this point, I would love to see AMP fallbacks for things like responsive menus, or the new modal showing up in the alpha version.

**Comment by robert1112** [comment]
Yes +10000 AMP will help boost SEO too. Please make it happen. Thank you.

**Comment by katsar0v** [comment]
Any update? Any AMP Plugin supported with Oxygen?

**Comment by ymarketing** [comment]
an AMP option or AMP compatibility to the official plugin could be nice too. https://wordpress.org/plugins/amp/

Would be great if Oxygen can add this, or create a new Oxygen AMP plugin addon.
